pine64/pinebook-pro/default.nix

```diff
@@ -0,0 +1,90 @@
+# This configuration file can be safely imported in your system configuration.
+{ config, pkgs, lib, ... }:
+
+{
+  nixpkgs.overlays = [
+    (import ./overlay.nix)
+  ];
+
+  boot.kernelPackages = lib.mkDefault pkgs.pinebookpro-manjaro-kernel;
```

Comment:
I would personally suggest making `linuxPackages_latest` the default, with documentation about using the Manjaro kernel, describing what benefits it provides.

Reply:
I was undecided about this point to be honest, a reason for my choice was that I don't know how fragile can be `linuxPackages_latest`. How likely is that when one update its `nixpkgs` the Pinebook Pro stops bootinkg/working?
Using a fixed and pinned version of the kernel is surely anti-pattern (compared to the Nix philosophy) but the chances of this possibility decrease. 

However I'll make this change if you deem it appropriate.

> (I guess mainly DP alt mode on type-c?)

It's the only benefit as far as I know but maybe there are other.